### PR TITLE
Adds a news links

### PR DESCRIPTION
**Pull request by GPrince00**
## Summary:

Added:

- A link for BitDegree
- A link for w3schools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ A collection of awesome useful and awesome links, resources and shiny things for
     * [Shaw Academy](https://www.shawacademy.com/)
     * [FrontendMasters](https://frontendmasters.com/)
     * [A Cloud Guru](https://acloudguru.com/)
+    * [BitDegree](https://www.bitdegree.org/)
+    * [w3schools](https://www.w3schools.com/)
```
Comment (Fcmam5) [inline README.md:49]: W3Schools is duplicated :) 

```suggestion
```

**Review by Fcmam5 — APPROVED** [review]
Thank you @GPrince00 for your contribution!

**Comment by GPrince00** [comment]
Can I add more if I find it?

**Comment by Fcmam5** [comment]
Yes please, that would be great

**Pull request merged**